webrtc.html

```diff
@@ -3045,10 +3038,21 @@ <h4>
                   </li>
                 </ol>
               </li>
+              <li>
```

Comment:
doesn't the data-tests tag follow along?

Reply:
Yes. I'm landing that test in https://phabricator.services.mozilla.com/D186256 today, so I might as well add it now. Thanks!